CHANGELOG.md

```diff
@@ -6,6 +6,63 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project does not yet adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 for setuptools_scm/PEP 440 reasons.
 
+## 1.2.0 Chia blockchain 2021-07-XX
+
+### Added
+
+- Portable pooled plots are now available using our new plot NFT. These allow you to plot new plots to an NFT that can either self farm or join and leave pools. During development, there were changes to the plot NFT so portable pool plots (those made with `-c` option to `chia plots create`) using pre-release code prior to this release are likely invalid on mainnet. OG plots made before this release can continue to be farmed side by side with the new portable pool plots but can not join pools using the official pooling protocol. You can learn more as a farmer by checking out the [pool user guide](https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide). Pool operators and those wanting to understand how the official pooling protocol operates should check out our [pooling implementation reference repository](https://github.com/Chia-Network/pool-reference).
```

Comment:
```suggestion
- Portable pooled plots are now available using our new plot NFT. These allow you to plot new plots to an NFT that can join and leave pools, allowing pooling or self farming operation. During development, there were changes to the plot NFT so portable pool plots (those made with the previously unsupported `-c` option to `chia plots create`) using code prior to this release will be invalid on mainnet. OG (non-pooling) plots made before this release can continue to be farmed side by side with the new portable pool plots but can not join pools using the official pooling protocol. You can learn more as a farmer by checking out the [pool user guide](https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide). Pool operators and those wanting to understand how the official pooling protocol operates should check out our [pooling implementation reference repository](https://github.com/Chia-Network/pool-reference).
```

Reply:
~~either self farm or join~~:  Just trying to emphasize that they can move from state to state, no matter their start state.

`will be invalid`: If they wrote their own contract, AND continue to use the old client, it will work, but it's such a niche possibility, it seems better to be unambiguous that NFTs that used the in-development code are not expected to work, even though `will be invalid` is technically too general.